base/Hero.js

```diff
@@ -61,6 +64,13 @@ export class Hero extends ControllableChar {
         this.trying_to_push = false;
         this.trying_to_push_direction = null;
         this.push_timer = null;
+        this.field_abilities_list = {};
+    }
+
+    set_field_psynergies(){
+        this.field_abilities_list.move = new MoveFieldPsynergy(game, data);
```

Comment:
Be careful when using the variables `game` and `data`. This is only working because I set them for debugging purposes in the index.js file. Use `this.game` and `this.data`.